.github/workflows/f5-cla.yml

```diff
@@ -0,0 +1,52 @@
+name: F5 CLA
+
+on:
+  issue_comment:
+    types:
+      - created
+  pull_request_target:
+    types:
+      - opened
+      - closed
+      - synchronize
```

Comment:
Why this event instead of just `pull_request`? And why when `closed`?

Reply:
For context I just copied the action that @TasoOneAsia created in https://github.com/nginxinc/nginx-prometheus-exporter and this is the default that they have in their docs. I think we don't need `closed` since we don't lock the PR after merge. And again I'm not sure why `reopened` is not in the list for example.

Reply:
@lucacome You're likely right that we no longer need to target that `closed` event, since we opted to not lock PR's after merge. 

The action targets the `synchronize` event to ensure that the prompted authors haven't changed since the "opened" event was fired.